### PR TITLE
fix boot_cpu_model.check_flag.cpu_host on Opteron_G5

### PR DESCRIPTION
**Pull request by XueqiangWei**
fix boot_cpu_model.check_flag.remove_flag on Opteron_G5

Signed-off-by: Xueqiang Wei <xuwei@redhat.com>

### CODE REVIEW

**Comment by XueqiangWei** [comment]
ID 1265129  
ID 1265105

File: qemu/tests/cfg/boot_cpu_model.cfg [hunk 1]
```diff
@@ -49,9 +49,13 @@
...
                     cpu_model_AuthenticAMD = "Opteron_G3"
-                    cpu_model_flags = ",-sse4a,-xsave,-aes,-sse4.2"
+                    cpu_model_flags = ",-sse4a,-xsave,-aes,-sse4.2,+ht,+sep"
+                    cpu_model_flags_AuthenticAMD = ",-monitor,+vme,+lahf_lm,+cmp_legacy"
                 - cpuid_7_0_ebx_feature:
                     cpu_model_flags = ",+fsgsbase"
                     cpu_model = "SandyBridge"
                 - cpu_host:
                     cpu_model = "host"
+                    cpu_model_flags = ",+ht,+fma,+bmi1"
```
Comment (suqinhuang) [inline qemu/tests/cfg/boot_cpu_model.cfg]: Why do we need to add these flags, does test pass with your update?

File: qemu/tests/cfg/boot_cpu_model.cfg [hunk 2]
```diff
@@ -49,9 +49,13 @@
...
+                    cpu_model_flags_AuthenticAMD = ",+vme,+mmxext,+fxsr_opt,+osvw,+cmp_legacy,-x2apic,-tsc_deadline_timer"
+
```
Comment (suqinhuang) [inline qemu/tests/cfg/boot_cpu_model.cfg]: the same as above

File: qemu/tests/cfg/boot_cpu_model.cfg (hunk 1 above)
Comment (XueqiangWei) [inline qemu/tests/cfg/boot_cpu_model.cfg]: @suqinhuang, yes, the test is passed with the update.

Comment (suqinhuang) [inline qemu/tests/cfg/boot_cpu_model.cfg]: Did you test in other host, what's the result if test on Intel host or host < Optern_G5

Comment (XueqiangWei) [inline qemu/tests/cfg/boot_cpu_model.cfg]: yes, I test it on Intel host. the test is passed. I modified the AMD related flags.  and not test on host < Optern_G5

Comment (suqinhuang) [inline qemu/tests/cfg/boot_cpu_model.cfg]: need have a test on the host without ht,+fma,+bmi1

Comment (XueqiangWei) [inline qemu/tests/cfg/boot_cpu_model.cfg]: ok, I will check it.

**Comment by xutian** [comment]
@XueqiangWei  any update of this PR ?

**Comment by XueqiangWei** [comment]
Hi @xutian close this PR, and resend a new one.

**Pull request closed without merging**